core/src/main/java/io/questdb/network/IODispatcherLinux.java

```diff
@@ -158,10 +158,12 @@ protected boolean runSerially() {
         }
 
         // process timed out connections
-        final long deadline = timestamp - idleConnectionTimeout;
-        if (pending.size() > 0 && pending.get(0, M_TIMESTAMP) < deadline) {
-            processIdleConnections(deadline);
-            useful = true;
+        if (idleConnectionTimeout > 0) {
```

Comment:
when configured `idelConnectionTimeout` is set to 0 or is negative, why don't we automatically set to Long.MIN_VALUE?. When this is the case we would not need an extra `if`